symbols/spring.svg

```diff
@@ -1,9 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-
-<svg xmlns="http://www.w3.org/2000/svg"
-   version="1.1"
-   width="5"
-   height="7">
-	<rect width="5" height="7" fill="white" />
-	<path d="M 4,1 1,1 1,4 3,4 3,5 1,5 1,6 4,6 4,3 2,3 2,2 4,2 4,1 z" fill="#588499" />
+<svg
+   xmlns="http://www.w3.org/2000/svg"
+   version="1.1">
+  <g
+     id="layer1"
+     transform="translate(0,-1036.3622)">
+    <path
+       transform="matrix(1.1666667,0,0,1.1666667,0,1033.6955)"
+       d="m 6,13 a 3,3 0 1 1 -6,0 3,3 0 1 1 6,0 z"
+       id="path4333"
+       style="fill:none;fill-opacity:1;stroke:#ffffff;stroke-width:0.60000002;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+       inkscape:connector-curvature="0" />
```

Comment:
These are some Inkscape leftovers in this file, as seen above - please make sure to save the file as plain SVG.